candidate_models/model_commitments/vonecornets.py

```diff
@@ -10,17 +10,29 @@
 
 class VOneCORnetCommitment(CORnetCommitment):
     def start_recording(self, recording_target, time_bins):
+        self.recording_target = recording_target
```

Comment:
is setting this field needed?

Reply:
Yes. I also made a change in the CORnetCommitment to set that field. That's because the naming of the layers differs from the typical CORnet and so it cannot extract the region recorded from the activations layer in the look_at_cached method